uint32 node_id
int32 distance_left     # 左侧磁条距离：-630mm-0
int32 distance_middle   # 中间磁条距离：-630mm-630mm
int32 distance_right    # 右侧磁条距离：0-630mm